apps/snippets/models.py

```diff
@@ -24,24 +51,34 @@ def __str__(self):
         return self.title
 
     panels = [
+        FieldPanel('title'),
         PageChooserPanel('link_page'),
-        FieldPanel('title')
+        StreamFieldPanel('subpages')
     ]
 
 
+class NavigationMenuItem(Orderable, MenuItem):
```

Comment:
@Rineee I find it weird that we have two models in their own tables here. I think either the `MenuItem` should be abstract or the `Orderable` and `parent` could just be in the `MenuItem`.
Buuut it's a bit late for that now unless I change the very first migration and then we would not only need to delete our local DBs, but also redo the one on the dev-server. Is that worth it or do we just keep the two models in two tables?

Reply:
@fuzzylogic2000 Oh yes, you are right, that is not needed! I like the solution with the abstract class. So from my side I would be in favour of removing the table, always better to clean up and at this point no one should have much data anyways.. but if you think its too much effort, I am of course also fine with it!

Reply:
Alright, I did that! So before merging we have to think about what to do to the dev site then. :)